Fix bug preventing file selection on iPhone 6 iOS 12.5

Coteh opened the pull request:
- Using the legacy build of PdfJS allows for the Select File button to open the file dialog on iPhone 6 12.5.4. It also fixes the Ontario Ministry of Health button and an error that appears due to PdfJS import when opening the app in dev mode.
- Fix selected files not being used when selected on iOS devices. Safari on these devices currently does not support "input" event on selected file change. "change" event should be used to support these devices. https://stackoverflow.com/a/58342757 https://caniuse.com/input-event

I do not have an Apple Developer account currently, so I cannot test Apple Wallet pkpass generation and backend verification. However, I can verify that the Save Photo feature continues to print the receipt as expected on both my iPhone (iPhone 6 12.5.4) and my Mac (Safari 14.1.1 (15611.2.7.1.6, 15611), Firefox 92.0.1 (64-bit), and Chrome 94.0.4606.61 (Official Build) (x86_64)).

edit: Did another test and noticed that on my iPhone 6, the photo is generated but does not save. I believe this is due to `file-saver` module - it should open a new window with the image upon call to `saveAs`, not sure how to solve this yet.

straxus commented:
@Coteh thank you very much for doing this! Definitely please let us know if you find a fix for the file-saver module issue - we probably need to make sure the whole flow works before we can merge iPhone 6 support, however I know we've gotten numerous requests for it so that will be very appreciated if we can get it working!

Coteh commented:
@straxus I was able to narrow it down to `html-to-image` library erroring out when parsing `&nbsp;` characters. I replaced it with a margin-left and the image was then able to generate fine using `file-saver`.

Also, I noticed that if `image/png` is in the `accept` attribute for file input, then my iPhone 6 won't let me select a PDF. I changed it to `.png` instead after reading [this SO post](https://stackoverflow.com/a/57163798) and it seemed to fix the problem. I also checked the iPhone 11 simulator I was using, which did not experience this, and it still accepts both PDFs and PNGs even after this change.

straxus commented:
@Coteh excellent, thank you very much for your work on this and for your help in improving our older iOS version support!

straxus commented on components/Form.tsx:
```diff
@@ -442,7 +442,7 @@ function Form(): JSX.Element {
                         <input type='file'
                                id='file'
-                               accept="application/pdf,image/png"
+                               accept="application/pdf,.png"
```
I'm actually kinda surprised that iOS is happy with a mix of MIME type and file extension like this, but I trust the testing that was done on older devices around this

Coteh replied:
Yeah it is kinda surprising this works. I’m not too sure why... my guess is there’s some bug with how Safari/WebKit is handling filetype hints.

I did notice on [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept) that you can technically mix and match different identifier types (ie. have some be MIME and some be extension) so I believe this should still work across all modern browsers.

straxus reviewed (approved):
Looks good to me! @billylo1 any concerns with merging this in and deploying?
